net/chrony/files/chrony.conf

```diff
@@ -5,3 +5,7 @@ logchange 0.5
 
 # Don't log client accesses
 noclientlog
+
+# set the system clock else the kernel will always stay in UNSYNC state
+rtcsync
+rtconutc
```

Comment:
Adding rtcsync to the default config is a good idea. But I'd suggest to remove the rtconutc directive. It does not do anything with rtcsync as the kernel is in control of the RTC, not chronyd. It's meant to be used with rtcfile, which is not supported in the package (compiled without support for RTC tracking).